addons/binding/org.openhab.binding.lightify/src/main/java/org/openhab/binding/lightify/handler/DeviceHandler.java

```diff
@@ -0,0 +1,146 @@
+package org.openhab.binding.lightify.handler;
+
+import org.eclipse.smarthome.core.library.types.DecimalType;
+import org.eclipse.smarthome.core.library.types.HSBType;
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.library.types.PercentType;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.openhab.binding.lightify.internal.link.LightifyLink;
+import org.openhab.binding.lightify.internal.link.LightifyLuminary;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+
+import static org.openhab.binding.lightify.internal.LightifyConstants.CHANNEL_ID_COLOR;
+import static org.openhab.binding.lightify.internal.LightifyConstants.CHANNEL_ID_DIMMER;
+import static org.openhab.binding.lightify.internal.LightifyConstants.CHANNEL_ID_POWER;
+import static org.openhab.binding.lightify.internal.LightifyConstants.CHANNEL_ID_TEMPERATURE;
+import static org.openhab.binding.lightify.internal.LightifyConstants.PROPERTY_ZONE_ID;
+import static org.openhab.binding.lightify.internal.LightifyConstants.THING_TYPE_LIGHTIFY_BULB;
+import static org.openhab.binding.lightify.internal.LightifyConstants.THING_TYPE_LIGHTIFY_ZONE;
+
+public class DeviceHandler extends BaseThingHandler {
+
+    public static final Set<ThingTypeUID> SUPPORTED_TYPES = Collections.unmodifiableSet(new HashSet<ThingTypeUID>() {
+        {
+            add(THING_TYPE_LIGHTIFY_BULB);
+            add(THING_TYPE_LIGHTIFY_ZONE);
+        }
+    });
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DeviceHandler.class);
+
+    public DeviceHandler(Thing thing) {
+        super(thing);
+    }
+
+    @Override
+    public void initialize() {
+        super.initialize();
+
+        Runnable statusUpdater = () -> {
+            GatewayHandler gatewayHandler = getGatewayHandler();
+            LightifyLink lightifyLink = gatewayHandler.getLightifyLink();
+            LightifyLuminary luminary = getLuminary();
+            if (luminary != null) {
+                lightifyLink.performStatusUpdate(luminary, this::updateState);
+            }
+        };
+
+        scheduler.scheduleWithFixedDelay(statusUpdater, 1, 60, TimeUnit.SECONDS);
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        LOGGER.info(channelUID.toString());
+        LOGGER.info("Command: " + command);
```

Comment:
1. Please combine those two log messages into one
2. Pelase change logging level to `debug`
3. Please use parametrized logging instead of string concatenation. E.g.
```
LOGGER.debug("Command: {}", command);
```